### PR TITLE
start proj

### PR DESCRIPTION
**Pull request by Andrij-Maksymyshyn**

### CODE REVIEW

File: errors/errorCodes.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  CREATED: 201,
```
Comment (chestnyh) [inline errors/errorCodes.js]: Is this an error code?

File: errors/errorCodes.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  NO_CONTENT: 204,
```
Comment (chestnyh) [inline errors/errorCodes.js]: Is this an error code

File: api/users/usersRouter.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const usersRouter = require("express").Router();
+
+const { validateUser } = require("./middlewares");
+const controllers = require("./controllers");
+
+usersRouter.get("/", (_, res) => {
+  res.send("Hello!");
+});
```
Comment (chestnyh) [inline api/users/usersRouter.js]: Do you really need this?

File: api/users/services/addUser.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const { hashPassword } = require("../../../services");
+const { User } = require("../../../models");
+
+const addUser = async userObject => {
+  const hashedPassword = await hashPassword(userObject.password);
```
Comment (chestnyh) [inline api/users/services/addUser.js:5]: Usually for user creation we use password and repeatPassword fields

File: index.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+app.use(cors());
+app.use(express.json());
+app.use("/", mainRouter);
+app.use("*", notFoundError);
+app.use(mainErrorHandler);
+
+const { PORT = 5000, DB_HOST } = process.env;
+
+mongoose
+  .set("debug", true)
```
Comment (chestnyh) [inline index.js]: This value should depends on env variable

**Review by chestnyh — APPROVED** [review]

File: errors/errorCodes.js (hunk 1 above)
Comment (Andrij-Maksymyshyn) [reply errors/errorCodes.js]: fixed

File: errors/errorCodes.js (hunk 2 above)
Comment (Andrij-Maksymyshyn) [reply errors/errorCodes.js]: fixed

File: api/users/usersRouter.js (hunk 3 above)
Comment (Andrij-Maksymyshyn) [reply api/users/usersRouter.js]: fixed

File: api/users/services/addUser.js (hunk 4 above)
Comment (Andrij-Maksymyshyn) [reply api/users/services/addUser.js:5]: fixed

File: index.js (hunk 5 above)
Comment (Andrij-Maksymyshyn) [reply index.js]: fixed

**Pull request merged**